src/source_clang.cc

```diff
@@ -639,7 +639,7 @@ void Source::ClangViewAutocomplete::autocomplete() {
   auto buffer=std::make_shared<Glib::ustring>(get_buffer()->get_text());
   auto iter=get_buffer()->get_insert()->get_iter();
   auto line_nr=iter.get_line()+1;
-  auto column_nr=iter.get_line_offset()+1;
+  auto column_nr=iter.get_line_index()+1;
```

Comment:
This commit actually fixes autocomplete after UTF-8 characters. For instance:
std::string str="æøå";
if(str=="æøå" && str.<- autocomplete now works here